src/components/AlertModal.vue

```diff
@@ -27,8 +33,10 @@ export default {
   props: {
     title: { type: String, required: true },
     body: { type: String, required: true },
+    secondRow: { type: String, default: '' },
```

Comment:
`secondBody`. Maybe we can allow the alert modal to accept the body as an array of strings instead?